modules/system/ServiceProvider.php

```diff
@@ -218,9 +218,15 @@ protected function registerMarkupTags()
                 'studly'         => ['Str', 'studly'],
                 'trans'          => ['Lang', 'get'],
                 'transchoice'    => ['Lang', 'choice'],
-                'md'             => ['Markdown', 'parse'],
-                'md_safe'        => ['Markdown', 'parseSafe'],
-                'md_line'        => ['Markdown', 'parseLine'],
+                'md'             => function($value) {
+                    return $value ? Markdown::parse($value) : null;
```

Comment:
@mjauvin Maybe something like this? `null` would fail `is_string`, and it would ensure that we're only passing strings to `parse()`.
```suggestion
                    return (is_string($value) && $value !== '') ? Markdown::parse($value) : '';
```